supplementary_style_guide/style_guidelines/links.adoc

```diff
@@ -23,8 +23,15 @@ Follow these guidelines when linking externally:
 * Always include link text. Bare URLs (URLs without explanatory information) are unhelpful because they do not provide adequate context about the link target.
 * Use hyperlinks unless the URL is an example URL or is otherwise inaccessible to users. For information about using addresses in examples, see the _IBM Style_ guide.
 * By default, links are followable and crawlable. Do not use the "nofollow" link option unless absolutely necessary.
-* Avoid links to external sites when possible. External links can change or be unreliable.
 * Avoid deep links (to a specific page or image from another company) to prevent an ongoing maintenance responsibility and to prevent bypassing a website's legal information.
+* Avoid links to external sites when possible. External links can change or be unreliable.
```

Comment:
```suggestion
* Avoid links to external sites when possible. Links to external sites can change or be unreliable, and customers might infer that Red Hat endorses the linked content, even if that is not the intent.
```

Reply:
This bullet point (about _external sites_) is in a complete section about _external links_. We need to distinguish between the two terms.

Reply:
Changed per your suggestion.